<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GiveBook</class>
 <widget class="QWidget" name="GiveBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px</string>
   </property>
   <property name="text">
    <string>Give Book</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSearch">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSearch">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>UserName</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Book_name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineBookName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxUserName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGiveBook">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Give Book</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
